funders/index.md

```diff
@@ -30,3 +30,125 @@ A full list is [here](https://en.wikipedia.org/wiki/Crown_Research_Institute).
 - [The Alfred P. Sloan Foundation](https://sloan.org/)
 - [Chan Zuckerberg Initiative](https://chanzuckerberg.com/)
 - [Ford Foundation](https://www.fordfoundation.org/)
+
+## Individual Funding Opportunities
+
+The below funding opportunities have been taken from the [open-funding](https://github.com/ralphtheninja/open-funding) github repository and are sub-set of funding specifically relevant to academia.
+
+### [`Protocol Labs Research`](https://github.com/protocol/research#private-retrieval)
+
+#### About
+
+We are looking for research and development of interactive private communication mechanisms.
+Commensurate with scope, up to USD 150k per award for research grants and USD 300k per award for implementation grants.
+Works we are interested in funding include those which:
+
+Explore new mechanisms for private communication (e.g. with cryptographic, information theoretic, or statistical basis)
+Relax the traditional ‘web’ assumptions of a single origin to engage with the possibilities of pre-distributed CDN or content-addressed data.
+Prototype the use of novel network-layer privacy technologies in real systems.
+
+#### Eligibility Criteria
+
+Implementation grant
+We expect that a team of 2-3 research engineers/developers working for 1-2 years will be sufficient to carry out the prototype development projects envisioned in this call, but we will consider other proposals. Renewals are possible.
+
+#### Application
+
+We encourage you to reach out to `research-grants@protocol.ai` or visit `#private-retrieval` in the Lodestar Discord if you’re considering applying or have any questions.
+
+#### Deadline
+
+N/A
+
+---
+
+### [`Chan Zuckerberg Initiative`](https://chanzuckerberg.com/eoss/)
```

Comment:
These all look like they should be on their own pages - what do you think about that idea?